src/ncp/ncp_base.hpp

```diff
@@ -337,6 +337,16 @@ class NcpBase
     void        HandleJoinerCallback(otError aError);
 #endif
 
+    static void HandleMlrRegResult_Jump(void *              aContext,
```

Comment:
```suggestion
    static void HandleMlrRegResult(void *              aContext,
```
Usually we use same method name for the static and non-static methods.

Reply:
I think we should leave the `_Jump` postfix due to other functions using it in this file (`HandleJamStateChange_Jump`, `HandleCommissionerPanIdConflict_Jump`, and others). I believe using the different rules for `HandleMlrRegResult` will be an unnecessary complication.